### PR TITLE
Fix typo on Life of a Transaction page

### LINKED ISSUE #4666: Typo/grammar error on 'Life of a Distributed Transaction' page
In [Life of a distributed transaction](https://www.cockroachlabs.com/docs/dev/architecture/life-of-a-distributed-transaction.html#resolving-write-intents) there is a paragraph with some breakage.  Looks like it was supposed to be a bulleted list item, but it's not part of a list, and results in some ungrammatical language in this paragraph (seems to start at " - Missing, "):

> The incoming transaction can only continue once the blocking transaction completes (i.e., commits or aborts). - Missing, the resolver consults the write intent's timestamp. If it was created within the transaction liveness threshold, it treats the transaction record as exhibiting the PENDING behavior, with the addition of tracking the push in the range's timestamp cache, which will inform the transaction that its timestamp was pushed once the transaction record gets created.

### PR DESCRIPTION
**Pull request by Shiwanee**
Hi @lnhsingh ,

I have included "MISSING" in the bulleted list. 

Thanks,
Shiwanee

Closes #4666

### CODE REVIEW

File: v19.1/architecture/life-of-a-distributed-transaction.md [hunk 1]
```diff
@@ -129,14 +129,10 @@ If an operation encounters a write intent for a key, it attempts to "resolve" th
...
+- `PENDING`, the new transaction attempts to "push" the write intent's transaction by moving that transaction's timestamp forward (i.e.,  ahead of this transaction's timestamp); however, this only succeeds if the write intent's transaction has become inactive.
+  	- If the push succeeds, the operation continues.
+  	- If this push fails (which is the majority of the time), this transaction goes into the [`TxnWaitQueue`](https:www.cockroachlabs.com/docs/stable/architecture/transaction-layer.html#txnwaitqueue) on this node. The incoming transaction can only continue once the blocking transaction completes (i.e., commits or aborts).
+- `MISSING`, the resolver consults the write intent's timestamp. If it was created within the transaction liveness threshold, it treats the transaction record as exhibiting the `PENDING` behavior, with the addition of tracking the push in the range's timestamp cache, which will inform the transaction that its timestamp was pushed once the transaction record gets created. If the write intent is older than the transaction liveness threshold, the resolution exhibits the `ABORTED` behavior.
```
Comment (Shiwanee) [inline v19.1/architecture/life-of-a-distributed-transaction.md:135]: 1.Should we write "If it was created within the transaction liveness threshold,....." and "If the write intent is older than the transaction liveness threshold, the resolution exhibits the `ABORTED` behavior. " as indented bullets?

2. I read the STAGING option at https://www.cockroachlabs.com/docs/stable/architecture/transaction-layer.html, should we include that here too?

The above 2 points apply to all four files that I have updated for this issue.

**Review by lnhsingh — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8284)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @lnhsingh and @Shiwanee)

___
*[v19.1/architecture/life-of-a-distributed-transaction.md, line 128 at r1](https://reviewable.io/reviews/cockroachdb/docs/8284#-MHMEClm7yJ5DpwyDlxr:-MHMEClm7yJ5DpwyDlxs:bdtsye4) ([raw file](https://github.com/cockroachdb/docs/blob/47681eb1bec52a22a7d3fc7fcd0597215ef2205a/v19.1/architecture/life-of-a-distributed-transaction.md#L128)):*
> ```Markdown
> #### Resolving Write Intents
> 
> If an operation encounters a write intent for a key, it attempts to "resolve" the write intent by checking the state of the write intent's transaction. If the write's intent's transaction record is...
> ```

I know this isn't part of your edit, but since you're here... can you change `If the write's intent's transaction...` to `If the write intent's transaction...`?
___
*[v19.1/architecture/life-of-a-distributed-transaction.md, line 135 at r1](https://reviewable.io/reviews/cockroachdb/docs/8284#-MHE8VDtocHhnz5pZP6z-r1-135:-MHM9DvQ20lQa58HVCtN:b53zswj) ([raw file](https://github.com/cockroachdb/docs/blob/47681eb1bec52a22a7d3fc7fcd0597215ef2205a/v19.1/architecture/life-of-a-distributed-transaction.md#L135)):*
Great points!

> 1.Should we write "If it was created within the transaction liveness threshold,....." and "If the write intent is older than the transaction liveness threshold, the resolution exhibits the ABORTED behavior. " as indented bullets?
Yes, this is a good idea. Go ahead and make that change!

> I read the STAGING option at https://www.cockroachlabs.com/docs/stable/architecture/transaction-layer.html, should we include that here too?
Yes, it seems like that should be included here as well. I'm not sure what happens if an operation encounters a write intent's transaction record that's in `STAGING` though-- @rmloveland do you know what should be included here?

**Comment by rmloveland** [comment]
@lnhsingh let's leave that STAGING bit out for now - a lot more info than that needs to be added here.  Long story but this page is kind of out of date with the current txn protocol (or, more out of date than the others?) and I would like to remove it.  It dupes a lot of content that is already elsewhere in the architecture docs, and needs updating or removal.

**Review by lnhsingh — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/8284)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @Shiwanee)

___
*[v19.1/architecture/life-of-a-distributed-transaction.md, line 135 at r1](https://reviewable.io/reviews/cockroachdb/docs/8284#-MHE8VDtocHhnz5pZP6z-r1-135:-MHMEoQ29gQAemZrhzrE:bqkkn5k) ([raw file](https://github.com/cockroachdb/docs/blob/47681eb1bec52a22a7d3fc7fcd0597215ef2205a/v19.1/architecture/life-of-a-distributed-transaction.md#L135)):*
<details><summary><i>Previously, lnhsingh (Lauren Hirata Singh) wrote…</i></summary><blockquote>

Great points!

> 1.Should we write "If it was created within the transaction liveness threshold,....." and "If the write intent is older than the transaction liveness threshold, the resolution exhibits the ABORTED behavior. " as indented bullets?
Yes, this is a good idea. Go ahead and make that change!

> I read the STAGING option at https://www.cockroachlabs.com/docs/stable/architecture/transaction-layer.html, should we include that here too?
Yes, it seems like that should be included here as well. I'm not sure what happens if an operation encounters a write intent's transaction record that's in `STAGING` though-- @rmloveland do you know what should be included here?

</blockquote></details>

Ok, per Rich's feedback, please leave `STAGING` out for now but go ahead and make the first edit. Thanks!

**Comment by lnhsingh** [comment]
LGTM, thank you for editing @Shiwanee! I will merge.

**Comment by Shiwanee** [comment]
@lnhsingh and @rmloveland ,
Thank you so much for your time and guidance to help me close this issue.
Shiwanee

**Pull request merged**